src/openapi/transport/auth.js

```diff
@@ -20,6 +20,11 @@ const DEFAULT_TOKEN_REFRESH_PROPERTY_NAME_EXPIRES = 'expiry';
 const DEFAULT_TOKEN_REFRESH_MARGIN_MS = 0;
 const DEFAULT_TOKEN_REFRESH_CREDDENTIALS = 'include';
 
+// Max request limits are used to block infinite loop of authorization requests after transport 401 errors, which my happen if given
+// endpoint for whatever reasons constantly returns 401 error status (despite correct fresh authorization token refresh).
+const DEFAULT_MAX_AUTH_ERRORS = 10;

```

Comment:
Add comment describing what specificly those defaults drive.  Probably could just copy jsdocs for related parameters.

Reply:
It would be also good to rename those consts and related variable to make clear distinction between failing auth request and failing specific endpoints with auth status code.